packages/eslint-plugin-next/lib/index.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
         '@next/next/no-typos': 1,
         '@next/next/no-duplicate-head': 2,
         '@next/next/inline-script-id': 2,
+        '@next/next/no-assign-module-variable': 2,
       },
```

Comment:
Higher up in this file, you'll need to add to `rules`:

`'no-assign-module-variable': require('./rules/no-assign-module-variable'),`